[{"exception": "error", "inner": [], "outer_pre": [], "outer_suf": [], "all": []}, {"exception": "Exception", "inner": [{"provider": {"package": "java.lang", "class": "Throwable", "method": "<init>", "filename": "Throwable", "line": ""}, "caller": {"package": "org.eclipse.core.runtime", "class": "CoreException", "method": "<init>", "filename": "CoreException", "line": ""}}, {"provider": {"package": "org.eclipse.core.runtime", "class": "CoreException", "method": "<init>", "filename": "CoreException", "line": ""}, "caller": {"package": "org.eclipse.jdt.core", "class": "JavaModelException", "method": "<init>", "filename": "JavaModelException", "line": ""}}, {"provider": {"package": "org.eclipse.jdt.core", "class": "JavaModelException", "method": "<init>", "filename": "JavaModelException", "line": ""}, "caller": {"package": "org.eclipse.jdt.internal.core", "class": "JavaProject", "method": "getResolvedClasspath", "filename": "JavaProject", "line": ""}}], "outer_pre": [], "outer_suf": [], "all": [{"provider": {"package": "java.lang", "class": "Throwable", "method": "<init>", "filename": "Throwable", "line": ""}, "caller": {"package": "org.eclipse.core.runtime", "class": "CoreException", "method": "<init>", "filename": "CoreException", "line": ""}}]}]